### PR TITLE
Search hidden links

### LINKED ISSUE #41: Search for hidden items
As most links we provide are not relevant to all students and therefore hidden it would be nice to find them through the search functionality at least.

Example: [med2.tum.sexy](https://med2.tum.sexy) is a valid redirect but only users who went through the source will know that it even exists.

### PR DESCRIPTION
**Pull request by joschahenningsen**
fixes #41 

I also removed some dead code, hope this is fine.

Test it on https://sexy.joschas.page/

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -386,4 +386,8 @@ li:hover .pin, li.hover .pin {
     .card {
         border-color: #e5e6e7;
     }
-}
\ No newline at end of file
+}
+
+#othersSection{
+    display: none;
```
Comment (kordianbruck) [inline css/custom.css:392]: because there is too many there?

**Review by kordianbruck — APPROVED** [review]

File: css/custom.css (hunk 1 above)
Comment (joschahenningsen) [reply css/custom.css:392]: Others are just all remaining uncategorized links. We could just display them but ultimately that's just a huge chunk of links almost nobody needs. Hiding them when not searching is a good compromise imo.

Comment (kordianbruck) [reply css/custom.css:392]: But then we can also just remove the section no?

Comment (joschahenningsen) [reply css/custom.css:392]: The section is new, I created it specifically for searching through it :D

Comment (joschahenningsen) [reply css/custom.css:392]: If you want me to further explain the rationale behind my changes we can hop on discord anytime :)

Comment (kordianbruck) [reply css/custom.css:392]: ah, upps I thought it was the SPECIAL section.

**Pull request merged**